files/build_templates/docker_image_ctl.j2

```diff
@@ -243,6 +250,11 @@ start() {
     echo "Creating new ${DOCKERNAME} container with HWSKU $HWSKU"
     {%- endif %}
 
+    {%- if docker_container_name == "swss" %}
+    # Obtain the vendor name
+    ASIC_VENDOR=`$SONIC_CFGGEN -y /etc/sonic/sonic_version.yml -v asic_type`
```

Comment:
Move this into inside swss container?

Reply:
The same as the previous one. I'd like call this once during docker creation instead of many times.